src/components/farmedList.vue

```diff
@@ -84,7 +84,10 @@ export default defineComponent({
   },
   computed: {
     farmedBlocksList(): FarmedBlock[] {
-      return this.$client?.data?.farming.farmed || []
+      const result = this.$client.data?.farming.farmed.filter((block) => {
+        return block.rewardAddr == this.rewardAddress
```

Comment:
@isSerge looks like this update doesn't know anything about votes, right? That is where the most of rewards will come from though. It is an unsigned extrinsic, see https://github.com/subspace/subspace/pull/485

Reply:
yes, we will have a discussion on how do we display rewards today